4-audit_edit.ipynb

```diff
@@ -614,16 +620,15 @@
    "id": "ad59b531-a9a5-4892-a911-8d347c62b8f3",
    "metadata": {},
    "source": [
-    "## A. Editing Individual Cases\n",
+    "### Step 1: Editing Individual Cases to Tune the Trainee\n",
     "\n",
-    "In this first section of Part 2, we will demonstrate how to edit the Trainee on a Case by Case basis. Editing a Case allows the user to modify or \"fix\" the behavior of the Trainee by the targeted editing of a few select cases. The user has complete control over all data in the Trainee, making it dynamic and quickly adjustable. Users do not need to worry about minor mistakes as the Trainee can be fine tuned with this method after training.\n",
+    "Here we demonstrate how to edit a Trainee one case at a time. Editing a case allows the user to modify or \"fix\" the behavior of the Trainee by the targeted editing of a one or more cases. The user has complete control over all data in the Trainee, making it dynamic and quickly adjustable. Users do not need to worry about minor mistakes as the Trainee can be fine-tuned with this method after training.\n",
     "\n",
-    "In our use case, the anomalous cases identified for the `Adult` dataset in Recipe `3-anomaly_detection.ipynb` represent possible Cases we want to edit. We noticed certain cases with unusual values for `capital-gains` like 99999 that look like they are nominal values representing other values, such as blanks. Editing Cases allows us to easily correct these minor issues for an otherwise valid Case post training. \n",
+    "In our use case, the anomalous cases identified for the `Adult` dataset in Recipe `3-anomaly_detection.ipynb` represent possible cases we want to edit. We noticed certain cases with unusual values for `capital-gains` like 99999 that look like they are nominal values representing other values, such as blanks. Editing Cases allows us to easily correct these minor issues for an otherwise valid case post-training. \n",
```

Comment:
did we decide on capitalized vs. uncapitalized Cases?

Reply:
I was definitely trying to uncapitalize them, I see I missed one here though.

Reply:
oh I missed a bunch in this recipe, thanks for the comment